### DISCUSSION TITLE
Missing the custom board (N32B) details

### DISCUSSION THREAD

**Discussion by williamchenghu** [Shik-Tech/N32B_HiRes#3] [Q&A] [answered]
Hey folks, from the `platformio.ini` it seems a custom board "N32B" was used. However, that [might reside](https://docs.platformio.org/en/stable/platforms/creating_board.html) in a folder outside the git control (e.g., .platformio/boards/N32B.json). Wonder if it were available anywhere, or what parameters were tweaked over the generic [`micro`](https://docs.platformio.org/en/stable/boards/atmelavr/micro.html#configuration) board?

**Comment by shikshik** [comment, marked as answer]
Hey @williamchenghu , we provide the custom board file for our customers who bought the kits or the preloaded pro micro board from us.
You can use any generic Arduino pro micro or Leonardo board configs. Make sure to update your local web editor with your board's IDs, in case you want to use the web editor.

**Reply by williamchenghu** [reply]
Thanks a lot for the clarification!

**Reply by shikshik** [reply]
@williamchenghu you're welcome.  You will probably want to check out the newest firmware:
https://github.com/Shik-Tech/N32B_Macros
